packages/react/src/components/Forms/Checkbox/index.spec.tsx

```diff
@@ -106,4 +112,13 @@ describe("<Checkbox/>", () => {
       document.querySelector(".c__checkbox__group.c__field.c__field--error"),
     ).toBeInTheDocument();
   });
+  it("renders with text and text right", async () => {
+    render(<Checkbox label="First name" rightText="Some text right" />);
+    expect(screen.getByText("First name")).toBeInTheDocument();
+    expect(screen.getByText("Some text right")).toBeInTheDocument();
+  });
+
+  it("checks that the console didn't print an error", async () => {
```

Comment:
Has we have a `spyError.mockRestore` inside a `afterAll` this test will always pass, no?

Reply:
My worry is also that if some day we need to create a test that needs on purpose to trigger an error we will not be able to do it with the approach you choose. I think it could be a more generic approach to isolate everything in a encapsulated test like:


```
it("renders without errors", async () => {
    render(<Checkbox label="First name" rightText="Some text right" />);
    expect(screen.getByText("First name")).toBeInTheDocument();
    expect(screen.getByText("Some text right")).toBeInTheDocument();
    expect(spyError).not.toHaveBeenCalled();
});
```

WDYT? The same goes for the other tests of this PR

Reply:
Yes you're right, not the best.
What about that (https://github.com/openfun/cunningham/pull/158/commits/9aaa4ed3ea213a509a3c4d95ae8dae1003ded96e), if another prop is added for sure it will be checked.

Reply:
> Has we have a `spyError.mockRestore` inside a `afterAll` this test will always pass, no?

No, maybe with `afterEach`, with `afterAll` it is release only 1 time, at the end of all the tests.